.tool-versions

```diff
@@ -1,3 +1,4 @@
 cf 7.1.0
 yarn 1.22.10
 nodejs 16.13.1
+ruby 3.2.2
```

Comment:
You should have the legacy version file configured to enable compatibility with rbenv users https://github.com/asdf-vm/asdf-ruby#ruby-version-file and not need this

Reply:
oh yeah, good point